app/src/main/java/com/newgogiyumandroid/MainActivity.kt

```diff
@@ -1,64 +1,121 @@
 package com.newgogiyumandroid
 
-
-import androidx.appcompat.app.AppCompatActivity
+import android.net.ConnectivityManager
+import android.net.NetworkInfo
 import android.os.Bundle
+import android.view.Menu
+import android.view.MenuInflater
+import android.view.MenuItem
+import android.view.View
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
 import androidx.recyclerview.widget.LinearLayoutManager
 import com.newgogiyumandroid.JsonParsingLists.FoodList
 import com.newgogiyumandroid.RecyclerView.CustomAdapter
 import kotlinx.android.synthetic.main.activity_main.*
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import okhttp3.*
 import org.json.JSONArray
-import java.net.URL
+import java.io.IOException
+
 
 class MainActivity : AppCompatActivity() {
 
-    var data: MutableList<FoodList> = mutableListOf()
+    companion object {
+        private const val MENU_URL = "https://gogiyum.com/api/menu"
+        var data: MutableList<FoodList> = mutableListOf()
```

Comment:
I think the MutableList<FoodList> is not static variable.

Reply:
Okay I will change it.